pcdsdevices/pump.py

```diff
@@ -202,6 +202,7 @@ class PTMPLC(Device):
     pump_at_speed = Cpt(EpicsSignalRO, ':AT_SPD_RBV', kind='normal')
     pump_accelerating = Cpt(EpicsSignalRO, ':ACCEL_RBV', kind='normal')
     pump_speed = Cpt(EpicsSignalRO, ':SPEED_RBV', kind='normal')
+    pump_current = Cpt(EpicsSignalRO, ':CURR_MON_RBV', kind='normal')
```

Comment:
https://github.com/pcdshub/lcls-twincat-vacuum/blame/11f5fca8a7e4ea5857bed510bcf4eba9c2a94c3e/L2SIVacuum/DUTs/Pumps/ST_PTM.TcDUT#L127-L134

From the above structure, it's been there a while so I'd imagine this won't be an issue for anything based on ST_PTM